src/components/LiquidityChartRangeInput/index.tsx

```diff
@@ -98,8 +98,10 @@ export default function LiquidityChartRangeInput({
   onRightRangeInput: (typedValue: string) => void
   interactive: boolean
   style?: CSSProperties
+  height?: string
 }) {
   const theme = useTheme()
+  const [ref, setRef] = useState<HTMLDivElement | null>(null)
```

Comment:
Because `ref= useRef()` has some delay to set the ref value. When it's settle done, it doesn't trigger the render again, so I need to make it a state to track whenever it's settle done to render and calc value again.